pkg/cmd/pulumi/package_extract_mapping.go

```diff
@@ -59,19 +58,28 @@ func newExtractMappingCommand() *cobra.Command {
 				return fmt.Errorf("no mapping found for key %q", key)
 			}
 
-			fmt.Printf("%s maps to provider %s\n", source, mapping.Provider)
+			fmt.Fprintf(os.Stderr, "%s maps to provider %s\n", source, mapping.Provider)
```

Comment:
This is why I originally made it require --out. It felt wrong to me to write out core information like this to stderr instead of stdout 😕 maybe this is better though even if it's writing primary output to the error stream

Reply:
Given how `pulumi package get-mapping` output looks in TF, it feels like this is a secondary piece of information.

Mapping data will always be structured to be machine readable (even if the method is plugin specific), so that feels like the primary piece of data to redirect.